template.php

```diff
@@ -681,6 +681,16 @@ function europa_preprocess_block(&$variables) {
       $variables['classes_array'][] = 'link-block';
       $variables['title_attributes_array']['class'][] = 'link-block__title';
       break;
+
+    case 'language_selector_site':
+      $variables['lang_code'] = $variables['elements']['code']['#markup'];
+      $variables['lang_name'] = $variables['elements']['label']['#markup'];
+      // Add class to block.
+      $link = url('splash');
+      $variables['classes_array'][] = 'lang-select-site';
+      $destination = drupal_get_destination()['destination'];
+      $variables['link'] = $link . '?' . $destination;
```

Comment:
Please change to 
$variables['link'] = url('splash') . '?' . drupal_get_destination()['destination'];